BLE_API.lib

```diff
@@ -0,0 +1 @@
+https://github.com/lancaster-university/BLE_API/#bbc2dc58f8da1d5a6d1882dddd5ce7399dc48889
```

Comment:
I assume these lib files are now needed for mbed-cli, is there any way around them?
If not, we now have two sources of truth, the yotta json file and these individual lib files.
What can we do to keep these in sync?
Can these lib files use git tags instead of commit hashes?

Reply:
They do not need to be inside the microbit-dal repository at all, they can be in the microbit repository or pulled in manually.

Reply:
Right but they have to be "somewhere", no?
If pulled manually does that mean extra steps when setting up the workspace?
I wouldn't mind having a script that parses the dependencies from the yotta json file and executes the right mbed-cli commands when setting up the workspace, what do you think the best place for a script like that would be?

Reply:
Somewhere yes.